lendingblock/component.py

```diff
@@ -1,7 +1,35 @@
+from abc import ABC
+from typing import List
 
 
-class Component:
-
+class Component(ABC):
```

Comment:
What's the benefit of using `ABC` here?

Reply:
Maybe none, but these classes should never be used directly, so why not mark them as such?